internal/router/router.go

```diff
@@ -434,6 +434,27 @@ func (r *Router) serveMetaGet(id *ResolvedURL, req *http.Request) (interface{},
 	return nil, errgo.WithCausef(nil, params.ErrNotFound, "unknown metadata %q", strings.TrimPrefix(req.URL.Path, "/"))
 }
 
+// GET id/meta/any?[include=meta[&include=meta...]]
+// https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetaany
+func (r *Router) serveMetaGetAny(id *ResolvedURL, req *http.Request) (interface{}, error) {
+	includes := req.Form["include"]
+	// If there are no includes, we have no handlers to generate
```

Comment:
This comment seems out of place now. As we have resolved the URL that will have worked out that the entity exists.